src/components/AssessmentCard/AssessmentRow.jsx

```diff
@@ -17,36 +18,41 @@ function AssessmentRow({ testNumber, gameName, skillTest, passingScore, numQuest
   if (numQuestions !== 0) {
     return (
       <tr>
-        <td>{testNumber}</td>
-        <td>{gameName}</td>
-        <td>{skillTest}</td>
+        <td className="test-number">{testNumber}</td>
+        <td className="game-name">{gameName}</td>
+        <td className="skill-test">{skillTest}</td>
         <td>{passingScore}</td>
-        <td>
+        <td className="player-score">
           <form onChange={handlePreScore}>
-            <input placeholder={0} type="number" name="Pre Score" min={0} max={numQuestions} />/
-            {numQuestions}
+            <input placeholder={0} type="number" name="Pre Score" min={0} max={numQuestions} />
+            {/*/
+            {numQuestions}*/}
```

Comment:
🚫 **[eslint]** <[spaced-comment](https://eslint.org/docs/rules/spaced-comment)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected space or tab before '*/' in comment.
```suggestion
            {numQuestions} */}
```